benchmarks/data/CCL_cosmoPar.dat

```diff
@@ -0,0 +1,15 @@
+ H0   70.001831054687500     
```

Comment:
What is this file used for?

Reply:
nothing, it was a file pushed to the repo with the benchmarks but we're not using it, I'll remove it.